CMakeLists.txt

```diff
@@ -22,6 +22,12 @@ cmake_minimum_required (VERSION 2.8)
 set( OPM_COMMON_ROOT "" CACHE PATH "Root directory containing OPM related cmake modules")
 option(SIBLING_SEARCH "Search for other modules in sibling directories?" ON)
 
+if(OPM_COMMON_ROOT AND opm-common_DIR)
```

Comment:
i vote we kill the _ROOT usage completely. it's legacy cmake stuff. the proper way to go about it is to use CMAKE_PREFIX_PATH entries.

Reply:
Fine by me. Sounds great. Will add a patch.